pkg/storage/compliance/tests.go

```diff
@@ -63,40 +63,37 @@ func testNotFound(t *testing.T, b storage.Backend) {
 func testListSuffix(t *testing.T, b storage.Backend) {
 	ctx := context.Background()
 
-	otherMod := "github.com/one/two-other"
-	mock := getMockModule()
-	err := b.Save(
-		ctx,
-		otherMod,
-		"v0.9.0",
-		mock.Mod,
-		mock.Zip,
-		mock.Info,
-	)
-	require.NoError(t, err, "Save for storage failed")
-	modname := "github.com/one/two"
-	versions := []string{"v1.1.0", "v1.2.0", "v1.3.0"}
-	for _, version := range versions {
-		mock := getMockModule()
-		err := b.Save(
-			ctx,
-			modname,
-			version,
-			mock.Mod,
-			mock.Zip,
-			mock.Info,
-		)
-		require.NoError(t, err, "Save for storage failed")
+	modVers := map[string][]string{
+		"github.com/one/two":       {"v1.1.0", "v1.2.0", "v1.3.0"},
+		"github.com/one/two/v2":    {"v2.1.0"},
+		"github.com/one/two/three": {"v0.9.0"},
```

Comment:
let's keep the old test of `github.com/one/two-other != github.com/one/two` to this new mix of scenarios.